app/src/androidTest/java/io/complicated/android_auth_scaffold/api/contact/ContactsClientUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package io.complicated.android_auth_scaffold.api.contact;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.test.mock.MockContext;
+
+import org.junit.Test;
+
+import okhttp3.HttpUrl;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+
+public class ContactsClientUnitTest {
+
+    @Test
+    public void getContactsUnitTest() throws Exception {
+
+        MockWebServer server = new MockWebServer();
```

Comment:
Everywhere that `final` keyword is applicable, can you add it? - I understand it's a nonstandard pattern, but it does reduce bugs and make things more clear (and potentially faster). - I'm careful to do this throughout all my Java code.